src/index.html

```diff
@@ -11,7 +11,183 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="div-with-input">
+          <label for="surname">Surname:</label>
+          <input
+            id="surname"
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="div-with-input">
+          <label for="name">Name:</label>
+          <input
+            id="name"
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="div-with-input">
+          <label for="age">How old are You?</label>
+          <input
+            id="age"
+            class="age"
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+
+        <div class="div-with-input">
+          <label for="birth">Full date of birth:</label>
```

Comment:
I would change this [for] att to be more descriptive sth like: **birth-date**.
Those attr does not add semantic to the page but for futhure you and your cooworkers it is better to name things as much descriptive as you can

Reply:
I renamed this name to `dateOfBirth`.